### PR TITLE
Updates for v3.1.7

### PR DESCRIPTION
**Pull request by lmm**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit df4853ae5622d18ccc2c9e7962eaf68d23048c4a

https://deploy-preview-2807--calico.netlify.com

File: _data/versions.yml [hunk 1]
```diff
@@ -1186,6 +1186,48 @@ v3.2:
       url: https://github.com/projectcalico/pod2daemon/releases/tag/v3.2.0
 
 v3.1:
+- title: v3.1.7
+  components:
+    calico/node:
+      version: v3.1.7
+      url: https://github.com/projectcalico/calico/releases/tag/v3.1.7
```
Comment (tmjd) [inline _data/versions.yml]: This tag does not exist

File: _data/versions.yml [hunk 2]
```diff
@@ -1186,6 +1186,48 @@ v3.2:
...
+    calicoctl:
+      version: v3.1.7
+      url: https://github.com/projectcalico/calicoctl/releases/tag/v3.1.7
+      download_url: https://github.com/projectcalico/calicoctl/releases/download/v3.1.7/calicoctl
+    calico/cni:
+      version: v3.1.7
+      url: https://github.com/projectcalico/cni-plugin/releases/tag/v3.1.7
+      download_calico_url: https://github.com/projectcalico/cni-plugin/releases/download/v3.1.7/calico
```
Comment (tmjd) [inline _data/versions.yml]: These download URLs are correct

File: _data/versions.yml [hunk 3]
```diff
@@ -1186,6 +1186,48 @@ v3.2:
...
+      download_calico_ipam_url: https://github.com/projectcalico/cni-plugin/releases/download/v3.1.7/calico-ipam
+    calico/kube-controllers:
+      version: v3.1.7
+      url: https://github.com/projectcalico/k8s-policy/releases/tag/v3.1.7
+    felix:
+      version: v3.1.7
+      url: https://github.com/projectcalico/felix/releases/tag/3.1.7
+    networking-calico:
+      version: 3.1.3
+      url: http://git.openstack.org/cgit/openstack/networking-calico/commit/?h=3.1.3
```
Comment (tmjd) [inline _data/versions.yml]: Not browse-able

File: _data/versions.yml (hunk 1 above)
Comment (lmm) [reply _data/versions.yml]: The release process for these versions has this tag pushed after this PR is merged.

File: _data/versions.yml (hunk 2 above)
Comment (lmm) [reply _data/versions.yml]: I just checked and the URL looks right to me - https://github.com/projectcalico/cni-plugin/releases/tag/v3.1.7

File: _data/versions.yml (hunk 3 above)
Comment (lmm) [reply _data/versions.yml]: I'll fix this as pointed out here https://github.com/projectcalico/calico/pull/2802/files#r316377866

**Pull request merged**